tests/test-cases/Group1-Docker-Commands/1-08-Docker-Logs.robot

```diff
@@ -192,9 +194,9 @@ Docker logs with timestamps and since certain time
     Should Match Regexp  ${date}  \\d{4}-\\d{2}-\\d{2}T\\d{2}:\\d{2}:\\d{2}\\.\\d{9}Z
```

Comment:
Commenting here since I can't seem to add a comment against line 178:
```
docker %{VCH-PARAMS} run busybox sh -c 'for i in $(seq 0 9) ; do sleep 1 && echo line $i; done'
```
should use `${busybox}`.